packages/ember-views/tests/views/view/remove_test.js

```diff
@@ -2,52 +2,16 @@ import { get } from 'ember-metal/property_get';
 import run from 'ember-metal/run_loop';
 import jQuery from 'ember-views/system/jquery';
 import View from 'ember-views/views/view';
-import ContainerView from 'ember-views/views/container_view';
+import { compile } from 'ember-template-compiler';
 
 import { registerKeyword, resetKeyword } from 'ember-htmlbars/tests/utils';
 import viewKeyword from 'ember-htmlbars/keywords/view';
 
 import { objectAt } from 'ember-runtime/mixins/array';
-
-
-// .......................................................
-// removeChild()
-//
+import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
 
 var parentView, child;
 var originalViewKeyword;
-QUnit.module('View#removeChild', {
```

Comment:
Should this really be removed or updated to work without using `Ember.ContainerView`?